src/gravel/tests/unit/controllers/test_services.py

```diff
@@ -0,0 +1,125 @@
+# project aquarium's backend
+# Copyright (C) 2021 SUSE, LLC.
```

Comment:
add the full license header?

Reply:
This file was moved back into place, so to keep the diff down I'll do it as a separate commit.